build_linux.sh

```diff
@@ -14,6 +14,8 @@ docker build -t tor-brave -f Dockerfile-linux \
     ${1+"$@"} .
 docker run --name tor-brave -d tor-brave
 docker cp tor-brave:/tor-$TOR_VERSION/install/bin/tor tor-$TOR_VERSION-linux-brave-$BRAVE_TOR_VERSION
+docker kill tor-brave
```

Comment:
```shell
docker stop -t 15 tor-brave || docker kill tor-brave
```
would probably be better.

If you do `docker run --rm` above, you also won't have to do `docker rm` here.